reactor-aeron-core/src/main/java/reactor/aeron/AeronResources.java

```diff
@@ -303,17 +303,36 @@ public AeronResources workerIdleStrategySupplier(Supplier<IdleStrategy> s) {
         });
   }
 
+  /**
+   * Shortcut method for {@code eventLoopGroup.next()}.
+   *
+   * @return {@code AeronEventLoop} instance
+   */
+  AeronEventLoop nextEventLoop() {
+    return eventLoopGroup.next();
+  }
+
+  /**
+   * Returns already used event loop. See {@code eventLoopGroup.first()}.
+   *
+   * @return {@code AeronEventLoop} instance
+   */
+  AeronEventLoop alreadyUsedEventLoop() {
```

Comment:
Rename it to the  "firstEventLoop"  to stay in sync with already est. convention on method   "nextEventLoop"  (and change javadoc accordingly)